exchange/bidder.go

```diff
@@ -115,16 +115,18 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 		}
 
 		if httpInfo.err == nil {
-			bids, moreErrs := bidder.Bidder.MakeBids(request, httpInfo.request, httpInfo.response)
+			bidResponse, moreErrs := bidder.Bidder.MakeBids(request, httpInfo.request, httpInfo.response)
 			errs = append(errs, moreErrs...)
-			for i := 0; i < len(bids); i++ {
-				if bids[i].Bid != nil {
-					bids[i].Bid.Price = bids[i].Bid.Price * bidAdjustment
+			if bidResponse != nil {
+				for i := 0; i < len(bidResponse.Bids); i++ {
+					if bidResponse.Bids[i].Bid != nil {
```

Comment:
Could you stick a `// TODO #280: Convert the bid price` comment here? Just so we don't lose track of it.

A `TODO #280` would also be useful in the `func (brw *bidResponseWrapper) validateBids().

Reply:
Yes good point, just done